controller/ProviderAdmin.php

```diff
@@ -41,4 +47,38 @@ public function getClassService()
     {
         return $this->getServiceLocator()->get(RdfLtiProviderRepository::class);
     }
+
+    public function editInstance()
```

Comment:
I protected method in the base controller, to provide a custom validator would enough, no? So you could remove the duplicated code in this class and also remove the class `LtiProviderForm`. Now if some default behavior changes in the 'editInstance' we are outdated.